draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -435,33 +477,52 @@
         <t>It is out of the scope of this document how the preference index is
         determined or configured.</t>
 
-        <t>The Preference Index Sub-TLV has the following format:</t>
+        <t>The Site Preference Index Sub-TLV has the following format:</t>
 
-        <figure anchor="Preference Index Sub-TLV"
-                title="Preference Index Sub-TLV">
+        <figure anchor="Site Preference Index Sub-TLV"
+                title="Site Preference Index Sub-TLV">
           <artwork><![CDATA[ 
    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
   |Site-Preference-Index Sub-Type | Length        | Reserved      |
   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                   Preference Index value                      |
+  |                Site Preference Index value                    |
   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ 
 	]]></artwork>
         </figure>
+	<!-- XXX JMH - The normative text below should indicate the field
+	lengths.  PDU diagrams disagreeing with the normative text is
+	unfortunately common in IETF and can lead to interop issues.
+
+	If the sub-tlv format is described above as discussed in a prior
+	comment, there is no need to re-state the sub-type and length sizes -
+	but also no harm in doing so. -->
```

Comment:
? The length field is there. Illustrated length = 5. is it acceptable?

Reply:
There are two comments here:
1. Normative text should describe the field lengths.  Don't make the users trust hand drawn diagrams.  Accidents in editing those diagrams tend to cause interop issues.
2. There's no need to repeatedly define the sub-tlv type and length fields if you consistently define them in the section that describes sub-tlvs.  See prior comment about such sub-tlv descriptions.

Reply:
How about just removing the sentence "The Length is 5 for the Site-Preference-Index Sub-Type."?

Reply:
The updated text lists the field lengths in each of the PDU components.  That's at least consistent.